### PR TITLE
remove management workspace

### PR DESCRIPTION
**Pull request by yubonluo**
### Description

remove management workspace

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx [hunk 1]
```diff
@@ -172,17 +171,10 @@ export const WorkspaceForm = ({
   const [color, setColor] = useState(defaultValues?.color);
   const [icon, setIcon] = useState(defaultValues?.icon);
   const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
-  const isEditingManagementWorkspace = defaultValues?.id === MANAGEMENT_WORKSPACE_ID;
 
   // feature visibility section will be hidden in management workspace
```
Comment (ruanyl) [inline src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx:175]: This comment should be removed

Comment (yubonluo) [reply src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx:175]: I will remove it right away

**Review by ruanyl — APPROVED** [review]

File: src/plugins/workspace/server/integration_tests/routes.test.ts [hunk 2]
```diff
@@ -233,7 +233,7 @@ describe('workspace service', () => {
         })
         .expect(200);
       // Global and Management workspace will be created by default after workspace list API called.
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/server/integration_tests/routes.test.ts:235]: Could you please remove the `Management` in the comment?

File: src/plugins/workspace/server/integration_tests/routes.test.ts [hunk 3]
```diff
@@ -296,7 +296,7 @@ describe('workspace service', () => {
         .expect(200);
       expect(findResult.body.total).toEqual(0);
       // Global and Management workspace will be created by default after workspace list API called.
```
Comment (SuZhou-Joe) [inline src/plugins/workspace/server/integration_tests/routes.test.ts:298]: Could you please remove the `Management` in the comment?

Comment (yubonluo) [reply src/plugins/workspace/server/integration_tests/routes.test.ts]: done

File: src/plugins/workspace/server/integration_tests/routes.test.ts (hunk 2 above)
Comment (yubonluo) [reply src/plugins/workspace/server/integration_tests/routes.test.ts]: done

**Review by SuZhou-Joe — APPROVED** [review]

**Pull request merged**